### DISCUSSION TITLE
netbox.circuits.circuits.create how do I get this to take custom fields.

### DISCUSSION THREAD

**Discussion by Bosborne96** [netbox-community/pynetbox#557] [Q&A] [answered]
I have tried performing a single circuit add with custom fields like below and have also created a Json object with the required and cf_name field names in order to test the bulk add. Both run and create the circuit, however, none of the custom fields are being populated. Let me know if I am missing an area where I need to present the custom values differently in either situation.

1.) netbox.circuits.circuits.create(required stuff=values, cf_<name>=value)

2.) I create a Json list of objects and 
netbox.circuits.circuits.create(json_list)

**Comment by markkuleinio** [comment, marked as answer]
When creating an object and populating the custom field, you do it with a dictionary like this:

```
device = netbox.dcim.devices.create(name="test", site_id=123, custom_fields={"field_name": "Field Contents"})
```

**Reply by Bosborne96** [reply]
Thanks, for the quick response!
